src/wp-admin/includes/upgrade.php

```diff
@@ -881,6 +881,11 @@ function upgrade_all() {
 	if ( $wp_current_db_version < 58975 ) {
 		upgrade_670();
 	}
+
+	if ( $wp_current_db_version < 60344 ) {
```

Comment:
Committers: When doing the commit to trunk this should be changed to the commit number in which it will appear. IE latest commit + 1.